build.gradle.kts

```diff
@@ -0,0 +1,130 @@
+plugins {
+    id("com.gradleup.shadow") version "8.+"
+    id("io.freefair.lombok") version "8.+"
+    id("maven-publish")
+    id("com.modrinth.minotaur") version "2.+"
+    id("xyz.wagyourtail.unimined") version "1.3.13"
+}
+
+val targetJava = JavaVersion.VERSION_17
+
+java {
+    sourceCompatibility = targetJava
+    targetCompatibility = targetJava
+    withSourcesJar()
+}
+
+base.archivesName.set(project.property("archives_base_name") as String)
+version = "${project.property("mod_version")}+${project.property("minecraft_version")}"
+group = project.property("maven_group") as String
+
+repositories {
+    maven("https://jitpack.io")
+    mavenCentral()
+}
+
+val lwjglVersion = "3.3.5"
+
+unimined {
+    minecraft {
+        ornitheMaven()
+        fabricMaven()
+
+        version("1.8.9")
+
+        merged {
+            legacyFabric {
+                loader("0.16.10")
```

Comment:
While yes the loader version may be updated I'd prefer for all properties to be in a single place still